test-cases/longevity/longevity-1TB-7days-authorization-and-tls-ssl.yaml

```diff
@@ -29,7 +29,7 @@ n_monitor_nodes: 1
 instance_type_db: 'i3.4xlarge'
 instance_type_loader: 'c5.4xlarge'
 
-nemesis_class_name: 'ChaosMonkey'
+nemesis_class_name: 'NodeRestartWithResharding'
```

Comment:
I think need to revert to ChaosMonkey

Reply:
PR  is not in status "ready for review" because of @roydahan wanted me to run 1TB longevity. So longevity yaml is not part of PR and will be reverted after I'll finish the test. So, this comment is not relevant